src/pages/CreateClass/index.js

```diff
@@ -192,7 +188,6 @@ function ToggleClassFormModal() {
         {canSpecifyFacilitator && (
           <Grid item md={3}>
             <DrawerLeft
-              pathwayID={pathwayID}
               setPathwayId={setPathwayId}
               setPathwayName={setPathwayName}
               setNewPathways={setNewPathways}
```

Comment:
You're not using `setNewPathways` in `DrawerLeft` component so you can remove it from here.